license-generator/generate.hs

```diff
@@ -65,6 +65,7 @@ depsNamesAndVersions = do
   excluded name =
     name == "purescript"
     || name == "purescript-ast"
+    || name == "purescript-cst"
```

Comment:
Similar here about licenses and `purescript-cst`. Marking this so we don't forget it.